Driver_FET.dcm

```diff
@@ -162,6 +162,18 @@ K Half Bridge Gate Driver
 F http://www.intersil.com/content/dam/Intersil/documents/hip4/hip4080a.pdf
 $ENDCMP
 #
+$CMP HIP4082_DIP
```

Comment:
Wildcarding the temp in the MPN will change the `_DIP` to `xB` and you can leave off the trailing `Z`.